### ISSUE TITLE
allow for running features NOT tagged with @tag

### ISSUE THREAD

**Issue by tknerr** [radish-bdd/radish#16]
Having a look at `radish -h` and browsing the sources of this repo, I found no way to express _"run all features_ **_NOT**_ _tagged with `@SomeTag`"_.

For example we would like to use: `radish *.feature --feature-tags Foo,Bar,-Baz` for running all features tagged with either "Foo" or "Bar", but not the ones tagged with "Baz"

Is that possible by now?

**Comment by timofurrer** [comment]
It's not possible at the moment. However, it seems interesting to me and I see the benefit you'd get..

Your suggestion leaves some open questions for me.
The argument `--feature-tags Foo,Bar,-Baz` and the according description with `Foo` **or** `Bar` **but not** `Baz` seems not obvious to me because it's like **or** and **and** conditions are mixed: The `Foo` and `Bar` references **all** passed features but the `-Baz` **only** the references the subsequent features. Did I get it wrong? Did you mean `or not the ones tagged with "Baz"`?
Do you see what I try to point out? Any ideas for how to solve it?

**Comment by timofurrer** [comment]
*push*

**Comment by BooleanCat** [comment]
I'm also interested in this feature. My use case is that I have a few particularly slow tests and I'd like to see all other tests pass whilst iterating on some feature without running the slow tests every time.

How would you feel about a separate flag rather than using the `-tag` method given above? Example: `radish --skip-feature-tags Bar,Baz`

I'm happy to do the work and submit a PR.

**Comment by timofurrer** [comment]
@BooleanCat Are you working on this one?

**Comment by BooleanCat** [comment]
Not currently but it's on my to do list - so likely soon! @timofurrer

**Comment by timofurrer** [comment]
I've implemented this feature with #42 

I went with a tag expression syntax. So you are able to do things like this:

```bash
radish test.feature --feature-tags 'regression'
radish test.feature --scenario-tags 'good_case and in_progress'
radish test.feature --scenario-tags 'good_case' --feature-tags 'regression'
radish test.feature --feature-tags 'regression and good_case and not real_hardware'
radish test.feature --feature-tags 'database or filesystem and bad_case'
```
